src/App.js

```diff
@@ -1,17 +1,42 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import ConferenceList from "./ConferenceList";
+import { getConferenceList } from "./api";
```

Comment:
This is a very minor comment, but have a think about how you want to structure your imports. It's nice to have an order to them. For example, alphabetical, and then order all of the packages first, then your internal imports, then css

Reply:
I think I've done this in the latest commit?